docs/integration.md

```diff
@@ -0,0 +1,129 @@
+
+    GET VALIDATOR_HOST:PORT/transfers/TRANSACTION_ID HTTP/1.1
+    
+>**VALIDATOR_HOST:PORT** - Host and port of any chosen validator
+**TRANSACTION_ID** - Transaction id of the transfer (Ex: ...)
+
+The response is in JSON format and contains the following data:
+```go
+    Recipient 		string 		`json:"recipient"`
+    RouterAddress 	string 		`json:"routerAddress"`
+    Amount 			string 		`json:"amount"`
+    NativeAsset 	string 		`json:"nativeAsset"`
+    WrappedAsset 	string 		`json:"wrappedAsset"`
+    Signatures 		[]string 	`json:"signatures"`
+    Majority 		bool 		`json:"majority"`
+```
+Property | Description
+---------- | ----------
+**Recipient** | EVM address of the receiver
+**RouterAddress** | Address of the router contract
+**Amount** | Transfer amount
+**NativeAsset** | Alias for the transferred asset
+**WrappedAsset** | Alias for the wrapped asset
+**Signatures** | Array of all provided signatures by the validators at the moment
+**Majority** | True if supermajority is reached and the transfer can be completed
+
+### 1.3 Claiming wrapped token
+When supermajority is reached only one step remains: for the user to claim their Wrapped version of HBAR or HTS token. In order to do that, the user must sign and submit a **mint transaction** to the Bridge Router Contract.
+
+The mint operation can be constructed using the following arguments:
+
+	mint(transactionId: bytes , wrappedAsset: address, receiver: address, amount: UInt256, signatures: bytes[] )
+
+Argument | Description
+---------- | ----------
+**transactionId** | The Hedera Transaction ID
+**wrappedAsset** | The corresponding wrappedToken contract address
+**receiver** | The address receiving the tokens
+**amount** | The desired minting amount
+**signatures** | The array of signatures from the members, authorising the operation
+
+### 1.4 Service fee distribution
+
+The main incentive for the Validators is the `service fee` paid by users. The fee is a percentage of the transferred amount, paid on the native asset. The Service fee is configurable property and determined by the validators.
+
+Fees are payed out from the Bridge account. Each Validator creates a Scheduled transaction and transfers the `service fee` amount from the Bridge account to the list of validators equally. Due to the nature of Scheduled Transactions, only one will be successfully executed, creating a scheduled Entity and all others will fail with `IDENTICAL_SCHEDULE_ALREADY_CREATED` error, and the transaction receipt will include the `ScheduleID` of the first submitted transaction. All validators, except the one that successfully created the Transaction execute `ScheduleSign` and once `n out of m` validators execute the Sign operation, the transfer of the fees will be executed.
+
+## 2. EVM-chain -> Hedera
+This functionality allows the user to transfer Wrapped HBAR or any supported by the bridge Wrapped Tokens from EVM-based chain to Hedera. 
+
+### 2.1 Supported wrapped tokens
```

Comment:
IMO it would be better to rename the section to:
`2. 1. Query supported tokens`